admin-dev/themes/default/js/bundle/product/product-combinations.js

```diff
@@ -217,6 +217,13 @@ var combinations = (function() {
           number.text(countSelectedProducts() + '/' + allProductCombination);
         });
 
+        $(function() {
```

Comment:
$(function() { ... }); is just jQuery short-hand for $(document).ready(function() { ... });
It's why the function is wrapper.